libs/stencil/src/schematics/storybook/init/init.spec.ts

```diff
@@ -3,7 +3,7 @@ import { Tree } from '@angular-devkit/schematics';
 import { getProjectConfig, readJsonInTree } from '@nrwl/workspace';
 import { createTestUILib, runSchematic } from '../../../utils/testing';
 
-describe('init', () => {
+fdescribe('init', () => {
```

Comment:
@DominikPieper Just wanted to ping you in case this change was unintended

Reply:
😦

Reply:
@twittwer Thanks, already re-enabled :-) And now working with 9.4.